plugins/chat/assets/javascripts/discourse/components/chat-message.js

```diff
@@ -517,8 +517,6 @@ export default class ChatMessage extends Component {
       this.currentUser.id
     );
 
-    this.args.forceRendering?.();
```

Comment:
This is not needed because a new reaction, or expanding/collapsing an image will change the size of the container